src/ansible/roles/dns/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Install dnsmasq package
```

Comment:
Mrack should get new dnsmasq output to generate dnsmasq config and this should go into shared playbooks so other projects can benefit from it as well.

Reply:
Well I thought it would make a sense to have the dns the same rol so it can be used also for container with a bit of tuning.

Reply:
We can do that in future but for now it make sense to have this here instead of having the dns setup copied in every metadata.